Navigation/MainStackNav.js

```diff
@@ -3,6 +3,13 @@ import { createStackNavigator, StackView } from '@react-navigation/stack';
 import * as React from 'react';
 import LoginPage from '../Pages/Login.js';
 import HomePage from '../Pages/Home.js';
+import ChildHomePage from '../Pages/ChildHomePage.js';
```

Comment:
Reformat top level directory

Reply:
can u specify further how you would like the directory to be reorganized? it's pretty comparable to the survivorFitness directory so def could use some clarification on how to improve it. thx!